README.md

```diff
@@ -137,7 +137,8 @@ print(result?.insertedID ?? "") // prints `.int64(100)`
 **Async**:
 ```swift
 let query: BSONDocument = ["a": 1]
-let result = collection.find(query).flatMap { cursor in
+let options = FindOptions(sort: ["_id": -1])
```

Comment:
I don't think it would be obvious from context what this is doing. maybe we could add a comment or something?

Reply:
done. I figure these examples will largely be used for copy/paste, so I included the comment in both examples to ensure it gets picked up regardless of how a user tries out the driver.